Formula/redpen.rb

```diff
@@ -1,9 +1,8 @@
 class Redpen < Formula
   desc "Proofreading tool to help writers of technical documentation"
   homepage "http://redpen.cc/"
-  url "https://github.com/redpen-cc/redpen/releases/download/redpen-1.10.3/redpen-1.10.3.tar.gz"
-  sha256 "0a7543e3961428ce68eb47c964e8988ec2f585db6e32ded582eb6626a98ffdd2"
-  revision 1
+  url "https://github.com/redpen-cc/redpen/releases/download/redpen-1.10.4/redpen-distribution-1.10.4-assembled.tar.gz"
```

Comment:
What's the difference between this and [the 1.10.4.tar.gz](https://github.com/redpen-cc/redpen/archive/redpen-1.10.4.tar.gz) that's also on the releases page that would match what we shipped last time round?